crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs

```diff
@@ -234,47 +236,29 @@ impl<'a> GenerateStage<'a> {
       }
     }
 
-    let index_sorted_cross_chunk_imports = index_cross_chunk_imports
-      .into_iter()
-      // FIXME: Extra traversing. This is a workaround due to `par_bridge` doesn't ensure order https://github.com/rayon-rs/rayon/issues/551#issuecomment-882069261
-      .collect::<Vec<_>>()
-      .into_par_iter()
-      .map(|cross_chunk_imports| {
-        let mut cross_chunk_imports = cross_chunk_imports.into_iter().collect::<Vec<_>>();
-        cross_chunk_imports.sort_by_cached_key(|chunk_id| {
-          let mut resource_ids = chunk_graph.chunks[*chunk_id]
-            .modules
-            .iter()
-            .map(|id| {
-              self.link_output.module_table.normal_modules[*id].resource_id.expect_file().as_str()
-            })
-            .collect::<Vec<_>>();
-          resource_ids.sort_unstable();
-          resource_ids
-        });
-        cross_chunk_imports
-      })
-      .collect::<Vec<_>>();
```

Comment:
The `cross_chunk_dynamic_imports` is collected by chunk modules `import_records`, the chunk modules is sorted, so the order is stable.
The `cross_chunk_imports` is from `chunk_meta_imports_vec`, it is collected by chunk modules `referenced_symbols`, the order is also stable.

Reply:
Didn't notice this part. Could you keep the original sorting logic. It have better stability on calculating chunk hash.